# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: supply-chain/config.toml
# Commit: 2b647eb982bc
# Full Hash: 2b647eb982bc0068d39fe58c6f59bebe55909343
# Author: Dana Keeler <dkeeler@mozilla.com>
# Date: 2023-06-07 21:43:58
# Regressor Bug: 1834862
# File Overlap Count: 1
# Description:
#   Bug 1834862 - implement nsICryptoHash in rust r=jschanck,supply-chain-reviewers
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D179727
# ==============================================================================

diff -r 7c11166969ff -r 2b647eb982bc supply-chain/config.toml
--- a/supply-chain/config.toml	Wed Jun 07 15:45:14 2023 +0000
+++ b/supply-chain/config.toml	Wed Jun 07 15:56:37 2023 +0000
@@ -777,10 +777,6 @@
 version = "0.9.2"
 criteria = "safe-to-deploy"
 
-[[exemptions.sha1]]
-version = "0.10.0"
-criteria = "safe-to-run"
-
 [[exemptions.sha2]]
 version = "0.10.2"
 criteria = "safe-to-deploy"